From 2432bbf56b943fbaa594ea6e8f57e8b99f654168 Mon Sep 17 00:00:00 2001
From: Sunil V L <sunilvl@ventanamicro.com>
Date: Thu, 3 Mar 2022 19:13:08 +0530
Subject: [RFC V1 4/5] uefi-sct/SctPkg: Install RISCV_EFI_BOOT_PROTOCOL test
 module

REF: https://bugzilla.tianocore.org/show_bug.cgi?id=3837

This patch enables installing the RISCV_EFI_BOOT_PROTOCOL test
cases into the SCT directory.

Signed-off-by: Sunil V L <sunilvl@ventanamicro.com>
---
 uefi-sct/SctPkg/CommonGenFramework.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/uefi-sct/SctPkg/CommonGenFramework.sh b/uefi-sct/SctPkg/CommonGenFramework.sh
index 779a6a44..fb9043b5 100755
--- a/uefi-sct/SctPkg/CommonGenFramework.sh
+++ b/uefi-sct/SctPkg/CommonGenFramework.sh
@@ -222,6 +222,10 @@ then
     cp $ProcessorType/PartitionInfoBBTest.efi                  $Framework/Test/ > NUL
     cp $ProcessorType/UFSDeviceConfigBBTest.efi                $Framework/Test/ > NUL
     cp $ProcessorType/ResetNotificationBBTest.efi              $Framework/Test/ > NUL
+    # Only include RiscVBootProtocolBBTest.efi if the file exists (true on RISCV64)
+    if [ -e $ProcessorType/RiscVBootProtocolBBTest.efi ]; then
+      cp $ProcessorType/RiscVBootProtocolBBTest.efi              $Framework/Test/ > NUL
+    fi
 
     # *********************************************
     # Copy ENTS binary
-- 
2.25.1

